app/templates/application.hbs

```diff
@@ -109,6 +109,57 @@
         {{#menu.menu-item
           title=layerGroup.model.title
           active=citymap}}
+          <ul class="legend no-bullet">
+            <li class="legend-item">
+              <svg class="legend-icon line-array" height="10" width="17" viewBox="0 0 17 10" preserveAspectRatio="xMinYMid">
```

Comment:
`legend-item` might be a good candidate for a component later. Let's do the others, look at ZoLa, and see if there's an abstraction here for later.